### DISCUSSION TITLE
Inquirer v9 is now an ECMAscript module

### DISCUSSION THREAD

**Discussion by SBoudrias** [SBoudrias/Inquirer.js#1126] [Announcements]
The [v9.0.0](https://github.com/SBoudrias/Inquirer.js/releases/tag/inquirer%409.0.0) is now a native Node esmodule.

Feel free to direct any issue upgrading to this thread and ask help from the community.

**Comment by Hacksore** [comment]
Probably a good idea to reference some helpful material for people to learn more about the difference between commonjs and es modules.

https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c

**Reply by SBoudrias** [reply]
Good call, I've updated the documentation here https://github.com/SBoudrias/Inquirer.js/commit/51bd91031dbeba529716c5c009f44941a2282570

Let me know/or send a PR if there's other useful guides we could link to! Cheers!

**Comment by SBoudrias** [comment]
If someone wants to help provide a common.js version, look at https://github.com/SBoudrias/Inquirer.js/issues/1159
